CHANGELOG.md

```diff
@@ -26,6 +26,9 @@ for a complete list of changes, enhancements, fixes and upgrade considerations.
 
  * Consumer.Close() previously did not trigger the final RevokePartitions
    callback, this is now fixed.
+ * AdminClient.CreateTopics() previously did not accept default value(-1) of
```

Comment:
This is in the v1.6.1 section of the CHANGELOG, but v1.6.1 has already been released so this change will go in the next release.
Create a new release section (call it v1.7.0) add this line there instead.